lib/commons/dom/get-target-rects.js

```diff
@@ -2,6 +2,7 @@ import findNearbyElms from './find-nearby-elms';
 import isInTabOrder from './is-in-tab-order';
 import { splitRects, hasVisualOverlap } from '../math';
 import memoize from '../../core/utils/memoize';
```

Comment:
This is a separate import on purpose as `memoize` when destructred from the utils import path causes problems for esbuild and it doesn't hoist the memoize function high enough in the build for functions to use it. We've gotten around that by importing it directly.